dbt/adapters/bigquery/connections.py

```diff
@@ -473,9 +476,14 @@ def execute(
             bytes_processed = query_job.total_bytes_processed
             processed_bytes = self.format_bytes(bytes_processed)
             message = f"{code} ({num_rows_formated} rows, {processed_bytes} processed)"
+            fields = query_table.schema
 
         response = BigQueryAdapterResponse(  # type: ignore[call-arg]
-            _message=message, rows_affected=num_rows, code=code, bytes_processed=bytes_processed
+            _message=message,
+            rows_affected=num_rows,
+            code=code,
+            bytes_processed=bytes_processed,
+            fields=fields,
```

Comment:
looks like you're introducing non-black formatted code. any chance you want to take formatting `imply.py` with `black`, to make your actual changes easier to diff here?

Reply:
Indeed I ran `black` after my changes but I definitely run it in a dedicated commit to push before mine.